src/services/jsDoc.ts

```diff
@@ -80,6 +80,17 @@ namespace ts.JsDoc {
         "virtual",
         "yields"
     ];
+    const syntaxKindDisplayPartFunctionDict = {

```

Comment:
I would name this `tagNameDisplayPart` -- you can look at the type to figure out that it's used as a dictionary, and it only maps tag names -- not syntax kinds for anything else.